multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -153,11 +153,9 @@ def export_onnx(
 
         return onnx_path
 
-    def export_tensorrt(
+    def optimize_for_inference(
```

Comment:
Support the provider argument to let users choose which provider to use.

Reply:
Added provider argument.